freetype/__init__.py

```diff
@@ -961,12 +962,12 @@ class Face( object ):
     FreeType root face class structure. A face object models a typeface in a
     font file.
     '''
-    def __init__( self, filename, index = 0 ):
+    def __init__( self, path_or_stream, index = 0 ):
         '''
         Build a new Face
 
-        :param str filename:
-            A path to the font file.
+        :param Union[str, io.BytesIO] path_or_stream:
```

Comment:
Why just BytesIO? One could well wish to pass an already open binary file object.
You could use typing.BinaryIO to catch both

Reply:
Hm, guess it doesn't hurt in a docstring.